packages/spec/renderer.js

```diff
@@ -16,19 +16,19 @@ describe('renderer', function () {
   });
 
   it('should create a renderer function', function () {
```

Comment:
Do you think a test for the newly introduced `bootstrapServer` feature could easily be added?